corehq/pillows/case.py

```diff
@@ -60,6 +65,58 @@ def get_case_to_elasticsearch_pillow(pillow_id='CaseToElasticsearchPillow', num_
     )
 
 
+def get_case_pillow(
+        pillow_id='case-pillow', ucr_division=None,
+        include_ucrs=None, exclude_ucrs=None,
+        num_processes=1, process_num=0, ucr_configs=None, skip_ucr=False,
+        processor_chunk_size=UCR_PROCESSING_CHUNK_SIZE, topics=None, **kwargs):
+    """
+    Return a pillow that processes cases. The processors include, UCR and elastic processors
+        Args:
+            skip_ucr: Can be set to True to avoid passing UCR processor, useful for tests
+    """
+    if topics:
+        assert set(topics).issubset(CASE_TOPICS), "This is a pillow to process cases only"
+    topics = topics or CASE_TOPICS
+    change_feed = KafkaChangeFeed(
+        topics, client_id=pillow_id, num_processes=num_processes, process_num=process_num
+    )
+    ucr_processor = ConfigurableReportPillowProcessor(
+        data_source_providers=[DynamicDataSourceProvider(), StaticDataSourceProvider()],
+        ucr_division=ucr_division,
+        include_ucrs=include_ucrs,
+        exclude_ucrs=exclude_ucrs,
+    )
+    if ucr_configs:
+        ucr_processor.bootstrap(ucr_configs)
+    case_to_es_processor = ElasticProcessor(
+        elasticsearch=get_es_new(),
+        index_info=CASE_INDEX_INFO,
+        doc_prep_fn=transform_case_for_elasticsearch
+    )
+    case_search_processor = get_case_search_processor()
+
+    checkpoint_id = "{}-{}-{}".format(
+        pillow_id, CASE_INDEX_INFO.index, case_search_processor.index_info.index)
+    checkpoint = KafkaPillowCheckpoint(checkpoint_id, topics)
+    event_handler = KafkaCheckpointEventHandler(
+        checkpoint=checkpoint, checkpoint_frequency=1000, change_feed=change_feed,
+        checkpoint_callback=ucr_processor
+    )
+    processors = [case_to_es_processor, case_search_processor, get_case_to_report_es_processor()]
```

Comment:
remove `get_case_to_report_es_processor` for enterprise clusters

Reply:
Updated, thanks.